### PR TITLE
Colors assets added and removed string based literals

### PR DESCRIPTION
**Pull request by onurhuseyincantay**
Added Colors asset and renamed static variables to that stored colors the pr has changes in the project file itself to be able to build it if you can revert it back and remove the bundle identifier and also rename the colors based on the design namings would be good :)

### CODE REVIEW

File: zeplin.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -618,15 +642,15 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 3;
-				DEVELOPMENT_TEAM = KPEKRP6RU9;
+				DEVELOPMENT_TEAM = CN7VBMZQZ3;
```
Comment (anonrig) [inline zeplin.xcodeproj/project.pbxproj]:
```suggestion
				DEVELOPMENT_TEAM = KPEKRP6RU9;
```

File: zeplin.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -597,15 +621,15 @@
...
 				INFOPLIST_FILE = zeplin/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.0.1;
-				PRODUCT_BUNDLE_IDENTIFIER = relevant.fruit.zeplin;
+				PRODUCT_BUNDLE_IDENTIFIER = relevant.fruit.zeplin1;
```
Comment (anonrig) [inline zeplin.xcodeproj/project.pbxproj]:
```suggestion
				PRODUCT_BUNDLE_IDENTIFIER = relevant.fruit.zeplin;
```

File: zeplin.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -618,15 +642,15 @@
...
```
Comment (anonrig) [inline zeplin.xcodeproj/project.pbxproj]:
```suggestion
				PRODUCT_BUNDLE_IDENTIFIER = relevant.fruit.zeplin;
```

File: zeplin/Sources/Screen/ScreenViewController.swift [hunk 4]
```diff
@@ -36,7 +36,7 @@ final class ScreenViewController: UIViewController, ios_toolkit.View, ErrorDispl
     // MARK: - Life cycle
     override func loadView() {
         view = viewSource
-        view.backgroundColor = UIColor(hex: 0x1d1d1d)
+      view.backgroundColor = Colors.windowBackgroundBlack.color
```
Comment (anonrig) [inline zeplin/Sources/Screen/ScreenViewController.swift]:
```suggestion
        view.backgroundColor = Colors.windowBackgroundBlack.color
```

File: zeplin/Sources/Splash/SplashView.swift [hunk 5]
```diff
@@ -15,7 +15,7 @@ final class SplashView: UIView {
         
         [logoImageView].forEach(addSubview(_:))
         
-        backgroundColor = UIColor(hex: 0x1d1d1d)
+      backgroundColor = Colors.windowBackgroundBlack.color
```
Comment (anonrig) [inline zeplin/Sources/Splash/SplashView.swift]:
```suggestion
        backgroundColor = Colors.windowBackgroundBlack.color
```

File: zeplin/Sources/Splash/SplashViewController.swift [hunk 6]
```diff
@@ -27,7 +27,7 @@ final class SplashViewController: UIViewController, View, ErrorDisplayer, Loadin
     // MARK: - Life cycle
     override func loadView() {
         view = viewSource
-        view.backgroundColor = UIColor(hex: 0x1d1d1d)
+      view.backgroundColor = Colors.windowBackgroundBlack.color
```
Comment (anonrig) [inline zeplin/Sources/Splash/SplashViewController.swift]:
```suggestion
        view.backgroundColor = Colors.windowBackgroundBlack.color
```

File: zeplin/Sources/Screen/ScreenCell.swift [hunk 7]
```diff
@@ -57,7 +57,7 @@ final class ScreenCell: UICollectionViewCell {
         super.init(frame: .zero)
         
         [verticalStack, overlay].forEach(addSubview(_:))
-        backgroundColor = UIColor(hex: 0x2f3033)
+      backgroundColor = Colors.profileButtonsBackground.color
```
Comment (anonrig) [inline zeplin/Sources/Screen/ScreenCell.swift]:
```suggestion
        backgroundColor = Colors.profileButtonsBackground.color
```

File: zeplin/Sources/Screen/ScreenCell.swift [hunk 8]
```diff
@@ -27,7 +27,7 @@ final class ScreenCell: UICollectionViewCell {
         view.textAlignment = .left
         view.textColor = .white
         view.font = .medium(12)
-        view.backgroundColor = UIColor(hex: 0x2f3033)
+      view.backgroundColor = Colors.profileButtonsBackground.color
```
Comment (anonrig) [inline zeplin/Sources/Screen/ScreenCell.swift]:
```suggestion
        view.backgroundColor = Colors.profileButtonsBackground.color
```

File: zeplin/Sources/Projects/ProjectsViewController.swift [hunk 9]
```diff
@@ -29,7 +29,7 @@ final class ProjectsViewController: UIViewController, View, ErrorDisplayer, Load
     // MARK: - Life cycle
     override func loadView() {
         view = viewSource
-        view.backgroundColor = UIColor(hex: 0x1d1d1d)
+      view.backgroundColor = Colors.windowBackgroundBlack.color
```
Comment (anonrig) [inline zeplin/Sources/Projects/ProjectsViewController.swift]:
```suggestion
        view.backgroundColor = Colors.windowBackgroundBlack.color
```

File: zeplin/Sources/Projects/ProjectCell.swift [hunk 10]
```diff
@@ -47,7 +47,7 @@ final class ProjectCell: UICollectionViewCell {
     private lazy var mainStack: UIStackView = .create(arrangedSubViews: [imageView, bottomContainer], axis: .vertical, alignment: .fill, distribution: .fillEqually, spacing: 0)
     private lazy var overlay: UIView = {
         let view = UIView()
-        view.backgroundColor = UIColor(hex: 0x2f3033)
+      view.backgroundColor = Colors.profileButtonsBackground.color
```
Comment (anonrig) [inline zeplin/Sources/Projects/ProjectCell.swift]:
```suggestion
        view.backgroundColor = Colors.profileButtonsBackground.color
```

File: zeplin/Sources/Projects/ProjectCell.swift [hunk 11]
```diff
@@ -14,17 +14,17 @@ final class ProjectCell: UICollectionViewCell {
         let view = UIImageView()
         view.kf.indicatorType = .activity
         view.contentMode = .scaleAspectFill
-        view.backgroundColor = UIColor(hex: 0x1d1d1d)
+      view.backgroundColor = Colors.windowBackgroundBlack.color
```
Comment (anonrig) [inline zeplin/Sources/Projects/ProjectCell.swift]:
```suggestion
        view.backgroundColor = Colors.windowBackgroundBlack.color
```

File: zeplin/Sources/Project/ProjectViewController.swift [hunk 12]
```diff
@@ -37,7 +37,7 @@ final class ProjectViewController: UIViewController, ios_toolkit.View, ErrorDisp
     // MARK: - Life cycle
     override func loadView() {
         view = viewSource
-        view.backgroundColor = UIColor(hex: 0x1d1d1d)
+      view.backgroundColor = Colors.windowBackgroundBlack.color
```
Comment (anonrig) [inline zeplin/Sources/Project/ProjectViewController.swift]:
```suggestion
        view.backgroundColor = Colors.windowBackgroundBlack.color
```

File: zeplin/Sources/Project/ProjectEditBarView.swift [hunk 13]
```diff
@@ -20,7 +20,7 @@ final class ProjectEditBarView: UIView {
     init() {
         super.init(frame: .zero)
         
-        backgroundColor = UIColor(hex: 0x2f3033)
+      backgroundColor = Colors.profileButtonsBackground.color
```
Comment (anonrig) [inline zeplin/Sources/Project/ProjectEditBarView.swift]:
```suggestion
        backgroundColor = Colors.profileButtonsBackground.color
```

File: zeplin/Sources/Profile/ProfileViewController.swift [hunk 14]
```diff
@@ -40,7 +40,7 @@ final class ProfileViewController: UIViewController, View, ErrorDisplayer, Loadi
     // MARK: - Life cycle
     override func loadView() {
         view = viewSource
-        view.backgroundColor = UIColor(hex: 0x1d1d1d)
+      view.backgroundColor = Colors.windowBackgroundBlack.color
```
Comment (anonrig) [inline zeplin/Sources/Profile/ProfileViewController.swift]:
```suggestion
        view.backgroundColor = Colors.windowBackgroundBlack.color
```

File: zeplin/Sources/Profile/ProfileView.swift [hunk 15]
```diff
@@ -74,14 +74,14 @@ final class ProfileView: UIView {
...
-        let label: UILabel = .create(text: texts, numberOfLines: 0, textAlignment: .center, textColor: UIColor(hex: 0xa1a2a3)!, font: .regular(11))
+      let label: UILabel = .create(text: texts, numberOfLines: 0, textAlignment: .center, textColor: Colors.descriptionGray.color, font: .regular(11))
         label.setLineSpacing(lineSpacing: 0, lineHeightMultiple: 1.2)
         return label
     }()
     
     private var separator: UIView = {
         let view = UIView()
-        view.backgroundColor = UIColor(hex: 0x545458)!.withAlphaComponent(0.65)
+      view.backgroundColor = Colors.seperatorGray.color
```
Comment (anonrig) [inline zeplin/Sources/Profile/ProfileView.swift]:
```suggestion
        view.backgroundColor = Colors.seperatorGray.color
```

File: zeplin/Sources/Profile/ProfileView.swift [hunk 16]
```diff
@@ -35,7 +35,7 @@ final class ProfileView: UIView {
         let view = UIButton(type: .custom)
         view.setTitle("Privacy Policy".localized(), for: .normal)
         view.setTitleColor(.white, for: .normal)
-        view.backgroundColor = UIColor(hex: 0x2f3033)
+      view.backgroundColor = Colors.profileButtonsBackground.color
```
Comment (anonrig) [inline zeplin/Sources/Profile/ProfileView.swift]:
```suggestion
        view.backgroundColor = Colors.profileButtonsBackground.color
```

File: zeplin/Sources/Notifications/NotificationsViewController.swift [hunk 17]
```diff
@@ -34,7 +34,7 @@ final class NotificationsViewController: UIViewController, View, ErrorDisplayer,
     // MARK: - Life cycle
     override func loadView() {
         view = viewSource
-        view.backgroundColor = UIColor(hex: 0x1d1d1d)
+      view.backgroundColor = Colors.windowBackgroundBlack.color
```
Comment (anonrig) [inline zeplin/Sources/Notifications/NotificationsViewController.swift]:
```suggestion
        view.backgroundColor = Colors.windowBackgroundBlack.color
```

File: zeplin/Sources/Login/LoginViewController.swift [hunk 18]
```diff
@@ -28,7 +28,7 @@ final class LoginViewController: UIViewController, View, ErrorDisplayer, Loading
     // MARK: - Life cycle
     override func loadView() {
         view = viewSource
-        view.backgroundColor = UIColor(hex: 0x1d1d1d)
+      view.backgroundColor = Colors.windowBackgroundBlack.color
```
Comment (anonrig) [inline zeplin/Sources/Login/LoginViewController.swift]:
```suggestion
        view.backgroundColor = Colors.windowBackgroundBlack.color
```

File: zeplin/Sources/Login/LoginView.swift [hunk 19]
```diff
@@ -20,7 +20,7 @@ final class LoginView: UIView {
         view.setTitleColor(.white, for: .normal)
         view.titleLabel?.font = UIFont.semiBold(16)
         view.cornerRadius = 24
-        view.backgroundColor = UIColor(hex: 0x419bf9)
+      view.backgroundColor = Colors.loginButtonBlue.color
```
Comment (anonrig) [inline zeplin/Sources/Login/LoginView.swift]:
```suggestion
        view.backgroundColor = Colors.loginButtonBlue.color
```

**Review by anonrig — CHANGES_REQUESTED** [review]
Thanks for the pull request. Other than the product identifier renaming issue, there are some several code styling issues.

**Comment by onurhuseyincantay** [comment]
@anonrig proved change requests and committed

**Review by anonrig — APPROVED** [review]

**Pull request merged**